phpstan-baseline.neon

```diff
@@ -1,4 +1,5 @@
 parameters:
+	reportUnmatchedIgnoredErrors: false
```

Comment:
this file should only be touched by tools
Are you sure this shouldnt be in the main neon file?